AirCasting/SessionViews/ThresholdSettings/ThresholdsSettingsView.swift

```diff
@@ -121,8 +121,12 @@ struct ThresholdsSettingsView: View {
 #if DEBUG
 struct HeatmapSettings_Previews: PreviewProvider {
     static var previews: some View {
-        ThresholdsSettingsView(thresholdValues: .constant([0, 20, 30, 40, 50]),
-                               initialThresholds: [])
+        ThresholdsSettingsView(thresholdValues: .constant(.init(array: [0, 20, 30, 40, 50])),
+                               initialThresholds: ThresholdsValue.init(veryLow:  10,
+                                                                       low:      15,
+                                                                       medium:   20,
+                                                                       high:     25,
+                                                                       veryHigh: 30))
```

Comment:
It won't compile because of this, I am going to just suggest a working solution if not make other yours 😄

```
static var previews: some View {
        ThresholdsSettingsView(thresholdValues: .constant(ThresholdsValue(veryLow: 10,
                                                                          low: 15,
                                                                          medium: 20,
                                                                          high: 25,
                                                                          veryHigh: 30)),
                               initialThresholds: ThresholdsValue.init(veryLow:  10,
                                                                       low:      15,
                                                                       medium:   20,
                                                                       high:     25,
                                                                       veryHigh: 30))
    }
```

Reply:
That's weird! It did compile, otherwise, I wouldn't make a PR :D I'm changing it like you suggested :)